ledger/ledger-api-bench-tool/README.md

```diff
@@ -17,6 +17,45 @@ bazel build //ledger/ledger-api-bench-tool:ledger-api-bench-tool_deploy.jar
 java -jar bazel-bin/ledger/ledger-api-bench-tool/ledger-api-bench-tool_deploy.jar --help
 ```
 
+## Configuration
+
+### Observer Parties
+
+You can specify the number of observer parties to create in the .yaml config as follows:
+```yaml
+submission:
+  num_observers: <NUMBER-OF-OBSERVERS>
+```
+This will tell the BenchTool to create `NUMBER-OF-OBSERVERS` parties.  
+The names of these parties will follow the pattern of `Obs-<INDEX>` where `0 <= INDEX < NUMBER-OF-OBSERVERS`
+and each party will have its own unique probability of `1 / 10^INDEX` for being selected as an observer of a contract.
+
+For example, when creating four observer parties they will be named `Obs-0`, `Obs-1`, `Obs-2`, `Obs-3`
+and their respective probabilities will be 100%, 10%, 1% and 0.1%.
+
+### Ledger Offset
+
+You can use any valid ledger offset.
+Additionally, you can use two special values `"ledger-begin"` and `"ledger-end"`
+
+### Consuming and Nonconsuming Exercises
+
+You can specify consuming and nonconsuming exercises to submit as below.
+```yaml
+submssion:
+  nonconsuming_exercises:
+    probability: 2.3
```

Comment:
I am sure if you asked a set of random people on the street of Moscow about it, 134% of them would tell you that probability must be between 0.0 and 1.0.
A bit nit-picking and it could go to a follow-up or be completely ignored, but how about using a different word for it like "incidence", "multiplier", "occurrence"

Reply:
Created a story in the epic not to forget it